scripts/private/_helpers.js

```diff
@@ -116,12 +116,14 @@ function areAddressesEqual(address1, address2) {
 }
 
 // Helper function to convert an address to a port ID
-function addressToPortId(portPrefix, address) {
+function addressToPortId(address, network) {
```

Comment:
we should discuss this, it was tricky to change given the updates to the port prefixes. Thinking of any way we can retrieve this where we don't have to update manually

Reply:
> Thinking of any way we can retrieve this where we don't have to update manually
Yep, that's why I changed this. If the version updates randomly it's hard to manage as you say. Which is why I've included a "clientSuffix" field in the registry e.g. https://github.com/polymerdao/polymer-registry/blob/main/chains/eip155%3A11155420.json#L39

So we keep the registry up-2-date + our dependency on the registry and this function would always return the right portId

Reply:
got it makes sense